mantidproject/fitbenchmarking
# Added Logging Functionality

#### Description of Work

Fixes #58 
Added logging functionality to the fitbenchmarking script by using python's logging module.

#### Testing Instructions

1. Run `example_runScripts.py` in the mantidpython terminal.
2. In `../fitbenchmarking/fitbenchmarking/logs` check if the three logging files have been created.
3. Check the logging files.

**Note:** Should improve this by catching the output of the mantid functions. Unfortunately, I had significant trouble doing that and created #67
